server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -352,8 +352,6 @@ public class ShuffleServerConf extends RssBaseConf {
   public static final ConfigOption<Long> FALLBACK_MAX_FAIL_TIMES =
       ConfigOptions.key("rss.server.hybrid.storage.fallback.max.fail.times")
           .longType()
-          .checkValue(
```

Comment:
The negative check is unnecessary

Reply:
Why do we remove this check?

Reply:
if -1，it means we could fallback directly once failed at the first time.

Reply:
What about the behaviour of `0`?

Reply:
For the first time fallback, it will reject for 0